crates/rattler/src/install/transaction.rs

```diff
@@ -46,6 +46,24 @@ pub enum TransactionOperation<Old, New> {
     Remove(Old),
 }
 
+impl<Old: Clone, New: Clone> TransactionOperation<&Old, &New> {
```

Comment:
Should we implement [`ToOwned`](https://doc.rust-lang.org/std/borrow/trait.ToOwned.html) instead?

Reply:
I don't think it would be possible to implement

```rust
impl<Old: Clone, New:Clone > Borrow<TransactionOperation<&Old, &New>> for TransactionOperation<Old, New> {
    fn borrow(&self) -> &TransactionOperation<&Old, &New> {
        // ???
    }
}
```

since we would have to create a new object and then return a reference to it.

which is required to implement `ToOwned`.